FPM.ftd

```diff
@@ -5,9 +5,14 @@ zip: github.com/fifthtry/ftd.dev/archive/refs/heads/main.zip
 language: en
 
 -- fpm.dependency: fifthtry.github.io/doc-site as ft
+
+-- fpm.dependency: fifthtry.github.io/product-switcher as bs
+
 -- fpm.dependency: fifthtry.github.io/product-switcher as ps
```

Comment:
```suggestion
-- fpm.dependency: fifthtry.github.io/product-switcher as ps
```

Don't add dependencies twice. Remove the first one. and try to build it locally.